earthmover/nodes/destination.py

```diff
@@ -113,8 +113,11 @@ def execute(self, **kwargs):
         self.size = os.path.getsize(self.file)
 
     def render_row(self, row: pd.Series):
-        row = row.astype("string").fillna('')
-        _data_tuple = row.to_dict()
+        row = row.to_dict()
+        types_to_cast = [bool, int, float]
+        keys_to_cast = list(filter(lambda x: type(row[x]) in types_to_cast, row.keys()))
+        row = dict(map(lambda x: (x[0], str(x[1]) if x[0] in keys_to_cast else (x[1] if x[1] else "")), row.items()))
```

Comment:
Would you mind adding a comment here? This is a pretty dense line and I'm struggling to grok what it's doing

Reply:
Thanks, I added a comment, not sure if that helps clarify?